projects/go-lib/src/lib/components/go-text-area/go-text-area.component.html

```diff
@@ -15,6 +15,8 @@
     [placeholder]="placeholder"
     [rows]="rows"
     [formControl]="control"
+    [attr.maxlength]="maxlength || null"
+    [attr.minlength]="minlength || null"
```

Comment:
One question I have about this is whether we should be using `attr.` for these. In the [Angular docs](https://angular.io/guide/binding-syntax) it says: "It is recommended that you set an element property with a property binding whenever possible."

Also [here](https://angular.io/guide/binding-syntax) it says "Attributes initialize DOM properties and then they are done. Property values can change; attribute values can't." I'm wondering if that means that when you use `attr.` it makes the binding static (i.e. it won't update if the value is changed)?

These two things make me think maybe we should just property binding here, unless the `text-area`s don't have these properties.